data/json/recipes/recipe_food.json

```diff
@@ -5184,7 +5421,7 @@
       [ "honey_bottled", 4 ],
       [ "honey_glassed", 4 ],
       [ "sugar", 40 ],
-      [ "syrup", 4 ],
+      [ "syrup", 3 ],
```

Comment:
Why maple syrup would be better than honey and other syrups? Consistency is better than favoring own additions.